.github/workflows/e2e.yaml

```diff
@@ -71,15 +73,19 @@ jobs:
           ./mvnw --batch-mode -Dcheckstyle.skip -Drat.skip -T2 -Dmaven.compile.fork -Dmaven.compiler.maxmem=3072 -DskipTests clean install
           ./mvnw --batch-mode -f test/e2e/pom.xml -pl e2e-base clean install
       - name: Cluster Tests (ES6/ZK/JDK8)
-        run: export E2E_VERSION=jdk8-1.3 && bash -x test/e2e/run.sh e2e-cluster/e2e-cluster-test-runner
+        run: export E2E_VERSION=jdk8-1.5 && bash -x test/e2e/run.sh e2e-cluster/e2e-cluster-test-runner --storage=elasticsearch
+      - name: Cluster Tests (InfluxDB/ZK/JDK8)
+        run: export E2E_VERSION=jdk8-1.5 && bash -x test/e2e/run.sh e2e-cluster/e2e-cluster-test-runner --storage=influxdb
       - name: Cluster With Gateway Tests (ES6/ZK/JDK8)
-        run: export E2E_VERSION=jdk8-1.3 && bash -x test/e2e/run.sh e2e-cluster-with-gateway/e2e-cluster-with-gateway-test-runner
+        run: export E2E_VERSION=jdk8-1.5 && bash -x test/e2e/run.sh e2e-cluster-with-gateway/e2e-cluster-with-gateway-test-runner
```

Comment:
Why this doesn't have `--storage=elasticsearch`? What is the default? Or what happens if no `--storage=`?

I expect a fail when missing this parameter.

Reply:
> Why this doesn't have `--storage=elasticsearch`? What is the default? Or what happens if no `--storage=`?

Not every e2e case is aware of the parameter `--storage`, `e2e-cluster-with-gateway` is one of them, it's intruduced recently and only `e2e-cluster` and `e2e-profile` are refactored then, IIRC

Reply:
Not add e2e-cluster-with-gateway test yet. Do we need it?

By the way, you are right, @kezhenxu94 .
(I am trying to add more testcases into e2e-ttl-influxdb, so I have not refactored it)

Reply:
I am OK with keeping the current status.